.github/workflows/pr_preview.yml

```
@@ -24,7 +24,7 @@ jobs:
       - uses: r-lib/actions/setup-r@v2
         with:
           r-version: 'release'
-          use-public-rspm: true
+          use-public-rspm: true        
```

```suggestion
          use-public-rspm: true
```

Ooops whitespace cruft